packages/salesforcedx-vscode-soql/package.json

```diff
@@ -67,6 +67,7 @@
     "test": "npm run test:vscode-integration",
     "test:vscode-integration": "node ../../scripts/run-vscode-integration-tests-with-top-level-extensions",
     "test:vscode-insiders-integration": "cross-env CODE_VERSION=insiders npm run test:vscode-integration",
+    "prevscode:package": "npx node-prune",
```

Comment:
I found that `npm prune --production` made no change in the size of the vsix.  So i didn't add it to the pre-package script.  

Below is my output of vsix sizes with 2 separate vsix's.  One created after npm prune and one before npm prune.  Note the sizes are the same.

```
-rw-r--r--  1 tcromer  wheel  7407284 Nov  2 11:59 salesforcedx-vscode-soql-50.3.0.after-prune.vsix
-rw-r--r--  1 tcromer  wheel  7407284 Nov  2 11:58 salesforcedx-vscode-soql-50.3.0.before-prune.vsix
```

Reply:
Posting so others can see this. We synced offline and the reason why this happens is because `npm prune --production` only removes devdependencies. I think you should still leave the prune step as part of building the vsix. Take a look at how it's done for the other extensions.